contracts/CrossChain.sol

```diff
@@ -38,6 +40,37 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
   // to prevent the utilization of ancient block header
   mapping(uint8 => uint64) public channelSyncedHeaderMap;
 
+
+  // BEP-171: Security Enhancement for Cross-Chain Module
+  bytes32 public constant EMERGENCY_SUSPEND_PROPOSAL = keccak256("EMERGENCY_SUSPEND_PROPOSAL");
+  bytes32 public constant REOPEN_PROPOSAL = keccak256("REOPEN_PROPOSAL");
+  bytes32 public constant CANCEL_TRANSFER_PROPOSAL = keccak256("CANCEL_TRANSFER_PROPOSAL");
+
+  uint16 public constant INIT_EMERGENCY_SUSPEND_THRESHOLD = 1;
+  uint16 public constant INIT_REOPEN_THRESHOLD = 2;
+  uint16 public constant INIT_CANCEL_TRANSFER_THRESHOLD = 2;
+
+  uint256 public constant EMERGENCY_PROPOSAL_EXPIRE_PERIOD = 1 hours;
```

Comment:
What about rename to `PROPOSAL_EXPIRE_PERIOD`? Since it is used for proposals, not only for emergency.

Reply:
keep `EMERGENCY_PROPOSAL`,  `EMERGENCY_SUSPEND` => `SUSPEND`